app/src/main/java/org/p2p/wallet/claim/model/HexTransaction.kt

```diff
@@ -0,0 +1,4 @@
+package org.p2p.wallet.claim.model
+
+@JvmInline
+value class HexTransaction(val transactionInHexString: String)
```

Comment:
i would rather extract it to :common / :core an make it just HexInstance / HexString for future reuse